packages/SwingSet/src/kernel/cleanup.js

```diff
@@ -1,39 +1,42 @@
-// import { kdebug } from '../lib/kdebug.js';
+// import { kdebug, onToggleDebug } from '../lib/kdebug.js';
 import { parseKernelSlot } from './parseKernelSlots.js';
 
+// let debugEnabled = false;
+// onToggleDebug(newEnableDebug => { debugEnabled = newEnableDebug; });
+
 export function getKpidsToRetire(kernelKeeper, rootKPID, rootKernelData) {
   const seen = new Set();
   function scanKernelPromise(kpid, kernelData) {
-    // kdebug(`### scanning ${kpid} ${JSON.stringify(kernelData)}`);
+    // debugEnabled && kdebug(`### scanning ${kpid} ${JSON.stringify(kernelData)}`);
```

Comment:
This `debugEnabled` flag is super ugly and doubly so in code that is commented out to begin with.  I don't have a problem with something much terser.